src/mantarray_desktop_app/process_monitor.py

```diff
@@ -112,8 +113,15 @@ def _check_and_handle_server_to_main_queue(self) -> None:
         except queue.Empty:
             return
 
+        msg: str
+        if "mantarray_nickname" in communication:
+            # Tanner (1/20/21): items in communication dict are used after this log message is generated, so need to creat a copy of the dict when redacting info
+            comm_copy = copy.deepcopy(communication)
+            comm_copy["mantarray_nickname"] = "*" * len(comm_copy["mantarray_nickname"])
```

Comment:
note for future--it'd probably be good to refactor out a function something like `create_redacted_str(unredacted:str)->str` and also use that in the file path redaction and the tests.  That way if for some reason we ever want to change the character to something other than an asterisk...or if for some reason we think it is too sensitive to use the actual length of the original string, and we just want it to always put 10 asterisks, there would just be one place to change